src/config.ts

```diff
@@ -222,7 +227,42 @@ export const ALLOW_PAGOPA_IP_SOURCE_RANGE = decodeCIDRs(
 export const API_KEY = getRequiredENVVar("API_KEY");
 export const API_URL = getRequiredENVVar("API_URL");
 
-export const API_CLIENT = new ApiClientFactory(API_KEY, API_URL);
+// whether we want to reuse sockets in the fetch client
+const isFetchKeepaliveEnabled = process.env.FETCH_KEEPALIVE_ENABLED === "true";
+
+// Returns a fetch instance backed by a keepalive-enabled HTTP agent
+const getKeepaliveFetch: () => typeof fetch = () => {
+  // see https://nodejs.org/docs/latest-v10.x/api/http.html#http_new_agent_options
+  const keepAliveMaxSockets =
+    process.env.FETCH_KEEPALIVE_MAX_SOCKETS === undefined
```

Comment:
this value must depend on nodejs processes, see

https://docs.microsoft.com/en-us/azure/app-service/app-service-web-nodejs-best-practices-and-troubleshoot-guide#my-node-application-is-making-excessive-outbound-calls

see also https://github.com/MicrosoftDocs/azure-docs/issues/8013